README.md

```diff
@@ -23,6 +23,18 @@ MySQL, Oracle, PostgreSQL and SQLite.
   so `date_format` will convert the mysql substitutions to the closest available sqlite substitutions.
   This means `date_format(field, '%b %D %Y') -> Jan 1st 2015` becomes `strftime('%m %d %Y', field) -> 01 01 2015`.
 
+> Note: Mysql's DATEADD, DATESUB have diverged a little from the original mysql functions:
+  `DATEADD(Date expression, Arithmetic expression, date unit string)`
+  `DATESUB(Date expression, Arithmetic expression, date unit string)`
+  For example:
+  `DATEADD(CURRENT_DATE(), 1, 'DAY')`
+
+> Note: Mysql's IFELSE works like this:
+  `IFELSE(Condition, If Expression, Else Expression)`
+  For example:
+  `IFELSE(field > 0, 'positive', 'not positive')`
+
+
```

Comment:
If these deviate then we should fix them rather than document them, what is the divergence?

Reply:
The differences:
# dateadd
Standard mysql: `DATE_ADD("2017-06-15", INTERVAL 10 DAY)`
Here:  `DATEADD(CURRENT_DATE(), 10, 'DAY')`
(datesub is the same pattern)
# ifelse
Standard mysql: `IF(1>2,2,3)`
Here: `IFELSE(1>2,2,3)`

I think changing the functions would break backward compatibility.

It caused me a few hours to figure out the differences and it seems others had the same issues:
https://stackoverflow.com/a/13802318/1665673